src/__tests__/lib/utils/hooks/SynapseAPI/useFavorites.test.tsx

```diff
@@ -33,12 +33,14 @@ const expected: PaginatedResults<EntityHeader> = {
       modifiedOn: 'earlier',
       createdBy: 'me',
       modifiedBy: 'you',
+      isLatestVersion: true,
     },
   ],
 }
 
-const SynapseClient = require('../../../../../lib/utils/SynapseClient')
-SynapseClient.getUserFavorites = jest.fn().mockResolvedValue(expected)
+const mockGetUserFavorites = jest
+  .spyOn(SynapseClient, 'getUserFavorites')
+  .mockResolvedValue(expected)
```

Comment:
Prefer `spyOn(...).mock` over assignment since it preserves types, and we don't need an ugly inline import.